### PR TITLE
update help

### PR DESCRIPTION
**Pull request by retokromer**
- list the actual parameters

### CODE REVIEW

File: searchlto [hunk 1]
```diff
@@ -16,7 +18,7 @@ _usage(){
     cat <<EOF
 $(basename "${0}") ${VERSION}
 
-Usage: $(basename "${0}") [Tape ID or Media ID] | -h
+Usage: $(basename "${0}") [-c] [-f] [-m] | -h
```
Comment (dericed) [inline searchlto]: The change here removes the hint that the search terms can be supplied as arguments, perhaps
```
Usage: $(basename "${0}") [-c] [-f] [-m] QUERY [QUERY...] | -h
```

Comment (retokromer) [reply searchlto]: Ah, thank you! I will update this also in the man page.

Do you think that the concise
```
Usage: $(basename "${0}") [-c] [-f] [-m] {QUERY} | -h
```
would be too cryptic?

Comment (dericed) [reply searchlto]: seems fine

Comment (retokromer) [reply searchlto]: fixed also in [man](https://github.com/amiaopensource/ltopers/pull/136)

**Pull request merged**